comms/lighthouse/src/peerjs-server/instance.ts

```diff
@@ -40,24 +40,24 @@ export const createInstance = ({
     config
   })
 
-  wss.on('connection', (client: IClient) => {
+  wss.on('connection', async (client: IClient) => {
     const messageQueue = realm.getMessageQueueById(client.getId())
 
     if (messageQueue) {
       let message: IMessage | undefined
 
       while ((message = messageQueue.readMessage())) {
-        messageHandler.handle(client, message)
+        await messageHandler.handle(client, message)
       }
       realm.clearMessageQueue(client.getId())
     }
 
     app.emit('connection', client)
   })
 
-  wss.on('message', (client: IClient, message: IMessage) => {
+  wss.on('message', async (client: IClient, message: IMessage) => {
```

Comment:
Same here, wss won't handle any error of the async function, nor wait for it in any scenario